<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Bell Clock -  by Afraz  :-)</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(81, 81, 81)</string>
   </property>
   <widget class="QPushButton" name="StartClockButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>247</y>
      <width>601</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255); background-color:rgb(107, 107, 107)</string>
    </property>
    <property name="text">
     <string>Start Clock (Standard Schedule)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>43</y>
      <width>681</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>45</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(150, 0, 0);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>HHS Bell Clock</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="EarlyStartClockButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>354</y>
      <width>601</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255); background-color:rgb(107, 107, 107)</string>
    </property>
    <property name="text">
     <string>Start Clock (Early Release)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="examStartClockButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>460</y>
      <width>601</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255); background-color:rgb(107, 107, 107)</string>
    </property>
    <property name="text">
     <string>Start Clock (Exam Schedule)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>236</y>
      <width>621</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 6px ridge rgb(116, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>698</y>
      <width>621</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 6px ridge rgb(116, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="EODMessageScreenButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>709</y>
      <width>601</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255); background-color:rgb(107, 107, 107);
</string>
    </property>
    <property name="text">
     <string>Add Reminders to Dismissal Message</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitappbutton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(107, 107, 107);
border: 4px groove rgb(130, 0, 0);
color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Quit App</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>266</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(150, 0, 0);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>Please Read!!!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>366</y>
      <width>411</width>
      <height>571</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(150, 0, 0);
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>-Standard Schedule is from 8:30 to 3:25

-Early Schedule is from 8:30 to 2:25

-Exam Schedule is from 8:30 to 12:40 and notifies students when exam 1 and 2 starts and ends

-The plain clock just shows time and weather, no notification

-When adding a reminder message, remember to click SAVE AND PREVIEW or else the message will not be saved</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="moreinfo">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>832</y>
      <width>601</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255); background-color:rgb(107, 107, 107);
</string>
    </property>
    <property name="text">
     <string>More Info...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>822</y>
      <width>621</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 6px ridge rgb(116, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="plainStartClockButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>567</y>
      <width>601</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255); background-color:rgb(107, 107, 107)</string>
    </property>
    <property name="text">
     <string>Start Clock (Plain Clock - No Schedule)</string>
    </property>
   </widget>
   <zorder>label_6</zorder>
   <zorder>label_2</zorder>
   <zorder>StartClockButton</zorder>
   <zorder>label</zorder>
   <zorder>EarlyStartClockButton</zorder>
   <zorder>examStartClockButton</zorder>
   <zorder>label_3</zorder>
   <zorder>EODMessageScreenButton</zorder>
   <zorder>quitappbutton</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>moreinfo</zorder>
   <zorder>plainStartClockButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
